tests/test_graphical_units/test_geometry.py

```diff
@@ -90,6 +110,32 @@ def construct(self):
         self.play(Animation(a))
 
 
+class DashedLineTest(Scene):
+    def construct(self):
+        a = DashedLine()
+        self.play(Animation(a))
+
+
+class CustomDashedLineTest(Scene):
+    def construct(self):
+        a = DashedLine(end=2 * DOWN, start=2 * UP, dash_length=0.5)
+        self.play(Animation(a))
+
+
+class TangentLineTest(Scene):
+    def construct(self):
+        circle = Circle(color=WHITE)
+        self.add(circle)
+        t1 = TangentLine(circle, 0.5, color=BLUE)
+        self.play(Animation(t1))
+        t2 = TangentLine(circle, 1.0, length=2, color=RED)
+        self.play(Animation(t2))
+        t3 = TangentLine(circle, 0.75, length=5, d_alpha=0.1, color=YELLOW)
+        self.play(Animation(t3))
+        t4 = TangentLine(circle, 0.321, length=1.5, d_alpha=-0.1, color=GREEN)
+        self.play(Animation(t4))
```

Comment:
```suggestion
       circle = Circle(color=WHITE)
        t1 = TangentLine(circle, 0.5, color=BLUE)
        t2 = TangentLine(circle, 1.0, length=2, color=RED)
        t3 = TangentLine(circle, 0.75, length=5, d_alpha=0.1, color=YELLOW)
        t4 = TangentLine(circle, 0.321, length=1.5, d_alpha=-0.1, color=GREEN)
        self.add(circle, t1, t2, t3, t4)
```
As we only test the last frame, I think this would be enough. I think why this self.play(Animation(abc)) was used before, was that otherwise, it would not work without any function to produce run time like wait or play, but this was solved in https://github.com/ManimCommunity/manim/pull/653

Reply:
Agreed. I can do it, but also I'd rather handle that on all tests in a separate PR. This PR is about CONFIG dicts.